src/Uno.UI/UI/Xaml/FrameworkElement.cs

```diff
@@ -160,16 +160,12 @@ public FlowDirection FlowDirection
 #if !SUPPORTS_RTL
 		[NotImplemented("__ANDROID__", "__IOS__", "__WASM__", "__MACOS__")]
 #endif
-		[GeneratedDependencyProperty(DefaultValue = FlowDirection.LeftToRight, Options = FrameworkPropertyMetadataOptions.Inherits, ChangedCallback = true)]
-		public static DependencyProperty FlowDirectionProperty { get; } = CreateFlowDirectionProperty();
-
-		private void OnFlowDirectionChanged(FlowDirection oldValue, FlowDirection newValue)
-		{
+		[GeneratedDependencyProperty(DefaultValue = FlowDirection.LeftToRight, Options =
 #if SUPPORTS_RTL
-			this.InvalidateArrange();
-			VisualTreeHelper.GetParent(this)?.InvalidateArrange();
```

Comment:
This seems like a different type of change. In practice we should not have to invalidate the parent, but maybe the original commit will tell why it was there?

Reply:
@jeromelaban It didn't work for me without that when I originally wrote this, and I haven't put enough time to understand why when I found this quite hacky way.

I was discussing this specific change with Martin. All we need is to verify that RTL checkbox still works as expected. If it does, then the change should be okay.